### ISSUE TITLE
handle object without events

### ISSUE THREAD

**Issue by Ambrosiani** [NordicMuseum/Wikimedia-Commons-uploads#28]
example of object with no events: https://digitaltmuseum.se/021027821725/skobotten

```
    "eventWrap": {
        "producers": [],
        "placesOfProduction": []
    },
```

error when running DiMuHarvester:

```
Traceback (most recent call last):
  File "importer/DiMuHarvester.py", line 862, in <module>
    main()
  File "importer/DiMuHarvester.py", line 855, in main
    harvester.load_collection(options.get('folder_id'))
  File "importer/DiMuHarvester.py", line 150, in load_collection
    self.process_single_object(item.get('artifact.uuid'))
  File "importer/DiMuHarvester.py", line 197, in process_single_object
    parsed_data = self.parse_single_object(data)
  File "importer/DiMuHarvester.py", line 272, in parse_single_object
    self.parse_event_wrap(data, raw_data.get('eventWrap'))
  File "importer/DiMuHarvester.py", line 709, in parse_event_wrap
    for event in event_wrap_data.get('events'):
TypeError: 'NoneType' object is not iterable
<class 'TypeError'>
CRITICAL: Closing network session.
```

relevant line of code: https://github.com/NordicMuseum/Wikimedia-Commons-uploads/blob/9c02cfbffe2085aac8106a3c5a5a7c60d530f8f1/importer/DiMuHarvester.py#L700

**Comment by Ambrosiani** [comment]
also fails at https://github.com/NordicMuseum/Wikimedia-Commons-uploads/blob/9c02cfbffe2085aac8106a3c5a5a7c60d530f8f1/importer/DiMuHarvester.py#L645

**Comment by Vesihiisi** [comment]
Just ran into this when working on nyfotograferad dräkt, so it's getting fixed in this batch.

**Comment by Vesihiisi** [comment]
Looks like this will be resolved by #29 (apart from line 645, so add that after merging)
